src/Features/Units/Connector.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Connect to the current unit system from Redux store
+ */
+import * as React from "react"
+
+import { UnitSystem } from "Features/Units/types"
+
+import { useUnitSystem } from "./hooks"
+
+interface Props {
+  children(props: RenderProps): JSX.Element
+}
+
+interface RenderProps {
+  unit_system: UnitSystem
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)